content/conditions/insect-bites-and-stings/sidebar-content.md

```diff
@@ -0,0 +1,12 @@
+!!! alert
```

Comment:
Is there a guideline on which one to use in which situation? Hernia page has a very similar message in the sidebar and it's using alert. Should it be consistent with that?